pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -27,6 +27,7 @@ stdenv.mkDerivation rec {
     "-DCMAKE_CXX_FLAGS=-std=c++11"
     "-DOpenMVG_BUILD_EXAMPLES=${if enableExamples then "ON" else "OFF"}"
     "-DOpenMVG_BUILD_DOC=${if enableDocs then "ON" else "OFF"}"
+    "-DOpenMVG_BUILD_SHARED=ON"
```

Comment:
Should we take `stdenv.hostPlatform.isStatic` into account?

Reply:
Good one, fixed